<h1 id="h_references">H_References<a aria-hidden="true" class="anchor-heading icon-link" href="#h_references"></a></h1>
<p>Organised in alphabetical order:</p>
<ol>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^95ccwy2tn3xk" href="#^95ccwy2tn3xk"></a>Allard, P.-M., Bisson, J. &#x26; Rutz, A. ISDB: In Silico Spectral Databases of Natural Products. Zenodo (2021). </li>
<li>
<h2 id=""><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^3kvwdyctonnv" href="#^3kvwdyctonnv"></a><a aria-hidden="true" class="anchor-heading icon-link" href="#"></a></h2>
Allard, P.-M. et al. Integration of Molecular Networking and In-Silico MS/MS Fragmentation for Natural Products Dereplication. Anal Chem 88, 3317–3323 (2016).</li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^56bhng6wvl4d" href="#^56bhng6wvl4d"></a>Barry A., T. Carboniferous Lepidodendraceae and Lepidocarpacea. ResearchGate (1978) doi:10.1007/BF02957853. </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^fzqjsn1h7hq4" href="#^fzqjsn1h7hq4"></a>Chambers, M. C. et al. A cross-platform toolkit for mass spectrometry and proteomics. Nat Biotechnol 30, 918–920 (2012). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^bc56ak014tej" href="#^bc56ak014tej"></a>Consortium, T. D. B. G. I. The Digital Botanical Gardens Initiative. Manubot <a href="https://digital-botanical-gardens-initiative.github.io/dbgi-green-paper/">https://digital-botanical-gardens-initiative.github.io/dbgi-green-paper/</a> (2022). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^mzjl22v177hk" href="#^mzjl22v177hk"></a>Consortium, T. E. The Earth Metabolome Initiative ... a brief presentation. (2022) doi:10.5281/zenodo.7486984.</li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^j5t0u383qi9x" href="#^j5t0u383qi9x"></a>Djoumbou Feunang, Y. et al. ClassyFire: automated chemical classification with a comprehensive, computable taxonomy. J Cheminform 8, 61 (2016). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^sex97gfkmrw2" href="#^sex97gfkmrw2"></a>Dührkop, K. et al. SIRIUS 4: a rapid tool for turning tandem mass spectra into metabolite structure information. Nat Methods 16, 299–302 (2019). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^jmetzeoa22lo" href="#^jmetzeoa22lo"></a>Dührkop, K. et al. Systematic classification of unknown metabolites using high-resolution fragmentation mass spectra. Nat Biotechnol 39, 462–471 (2021). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^4m68ue3rs0cy" href="#^4m68ue3rs0cy"></a>Fairon-Demaret, M. Some uppermost Devonian megafloras: a stratigraphical review. asgb (1986). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^3p0q0bvyjtbc" href="#^3p0q0bvyjtbc"></a>Guralnick, R. P. et al. Community Next Steps for Making Globally Unique Identifiers Work for Biocollections Data. Zookeys 133–154 (2015) doi:10.3897/zookeys.494.9352.  </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^gyjd2mvhbdtp" href="#^gyjd2mvhbdtp"></a>Hallé, F. Aux origines des plantes. vol. Vol. 1 (Fayard, Paris, 2008). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^d46zf9idf9g7" href="#^d46zf9idf9g7"></a>Lund, H. G. Definitions of  Tree and Shrub. ResearchGate <a href="https://www.researchgate.net/publication/281743733_Definitions_of_&#x27;tree&#x27;_and_&#x27;shrub&#x27;">https://www.researchgate.net/publication/281743733_Definitions_of_'tree'_and_'shrub'</a> (2015) doi:10.13140/RG.2.1.4103.9848. </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^eslokokmo0rs" href="#^eslokokmo0rs"></a>Meyer-Berthaud, B., Scheckler, S. E. &#x26; Wendt, J. Archaeopteris is the earliest known modern tree. Nature 398, 700–701 (1999). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^jut75dwyjv55" href="#^jut75dwyjv55"></a>Myers, O. D., Sumner, S. J., Li, S., Barnes, S. &#x26; Du, X. One Step Forward for Reducing False Positive and False Negative Compound Identifications from Mass Spectrometry Metabolomics Data: New Algorithms for Constructing Extracted Ion Chromatograms and Detecting Chromatographic Peaks. Anal. Chem. 89, 8696–8703 (2017). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^ktra0w22cs9d" href="#^ktra0w22cs9d"></a>Pluskal, T., Castillo, S., Villar-Briones, A. &#x26; Oresic, M. MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data. BMC Bioinformatics 11, 395 (2010). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^fuce5gbkvsxx" href="#^fuce5gbkvsxx"></a>Raven, J. &#x26; Crane, P. Trees. Current Biology 17, R303–R304 (2007). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^o885ohckztyp" href="#^o885ohckztyp"></a>Rutz, A. et al. Taxonomically Informed Scoring Enhances Confidence in Natural Products Annotation. Front. Plant Sci. 10, (2019). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^fktchxtnov0t" href="#^fktchxtnov0t"></a>Rutz, A. et al. The LOTUS initiative for open knowledge management in natural products research. Elife 11, e70780 (2022). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^d7ia8j3v0iru" href="#^d7ia8j3v0iru"></a>Unifr. A propos | Jardin botanique de l’Université de Fribourg |. Unifr about JBUF <a href="https://www.unifr.ch/jardin-botanique/fr/about/">https://www.unifr.ch/jardin-botanique/fr/about/</a>. </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^3jcvikm91uoh" href="#^3jcvikm91uoh"></a>Wang, M. et al. Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking. Nat Biotechnol 34, 828–837 (2016). </li>
<li><a aria-hidden="true" class="block-anchor anchor-heading icon-link" id="^d66eman67xtj" href="#^d66eman67xtj"></a>Kim, H. et al. NPClassifier: A Deep Neural Network-Based Structural Classification Tool for Natural Products | Journal of Natural Products. <a href="https://pubs.acs.org/doi/10.1021/acs.jnatprod.1c00399">https://pubs.acs.org/doi/10.1021/acs.jnatprod.1c00399</a>. </li>
</ol>
<hr>
<strong>Backlinks</strong>
<ul>
<li><a href="/dendron-dbgi/notes/j0dw5yx7u1t2uyitabax2h8">A_Contents</a></li>
<li><a href="/dendron-dbgi/notes/it2hbdgf7o7mejku2hwo817">B_Introduction</a></li>
<li><a href="/dendron-dbgi/notes/eu1plal3d6wub84ws6fp1gt">C_Methods</a></li>
<li><a href="/dendron-dbgi/notes/t942xq2f8o2247mgu5700pr">D_Metabolomic Analysis</a></li>
</ul>