<h1 id="17">17<a aria-hidden="true" class="anchor-heading icon-link" href="#17"></a></h1>
<h1 id="this-is-luca-cappellettis-dbgi-daily-open-notebook">This is Luca Cappelletti's DBGI daily open-notebook.<a aria-hidden="true" class="anchor-heading icon-link" href="#this-is-luca-cappellettis-dbgi-daily-open-notebook"></a></h1>
<p>Today is 2023.07.18</p>
<h2 id="todo-today">Todo today<a aria-hidden="true" class="anchor-heading icon-link" href="#todo-today"></a></h2>
<ul>
<li>Verify whether <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9372416/">BioLink</a> comes with the support of biological samples.</li>
</ul>
<h2 id="done">Done<a aria-hidden="true" class="anchor-heading icon-link" href="#done"></a></h2>
<h3 id="on-biolink">On Biolink<a aria-hidden="true" class="anchor-heading icon-link" href="#on-biolink"></a></h3>
<p><a href="https://www..com/in/deepakunni3/">Deepak Unni</a>, who is the main author of <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9372416/">BioLink</a>, said the following regarding the support of BioLink for biological samples:</p>
<blockquote>
<p>Biolink Model does have a few concepts that can be used to represent information about samples. But I don't think it is sufficiently modelled to support processing of samples as it is done for genomic sequencing or single cell sequencing. Ideally, it would be good to explore how the model can be extended to support this knowledge.</p>
</blockquote>
<p>Deepak also pointed me to the following resources:</p>
<ul>
<li><a href="https://github.com/microbiomedata/nmdc-schema">NMDC</a>: The NMDC is a multi-organizational effort to integrate microbiome data across diverse areas in medicine, agriculture, bioenergy, and the environment. This integrated platform facilitates comprehensive discovery of and access to multidisciplinary microbiome data in order to unlock new possibilities with microbiome data science.</li>
<li><a href="https://github.com/isamplesorg/metadata">iSamples</a>: schema for documenting physical samples</li>
<li><a href="https://github.com/EbiEga/ega-metadata-schema">EGA/ENA</a>: <a href="https://ega-archive.org/submission/quickguide">European genome-phenome archive</a> metadata schema.</li>
<li><a href="https://github.com/ghga-de/ghga-metadata-schema">GHGA</a>: Metadata schema for the German Human Genome-Phenome Archive</li>
<li><a href="https://github.com/fairgenomes/fairgenomes-semantic-model">FAIR Genomes</a>: Genomes semantic metadata model</li>
</ul>
<h3 id="why-having-a-hierarchy-for-the-earth-metabolome-kgs-node--edge-labels-is-important">Why having a hierarchy for the Earth Metabolome KG's node &#x26; edge labels is important<a aria-hidden="true" class="anchor-heading icon-link" href="#why-having-a-hierarchy-for-the-earth-metabolome-kgs-node--edge-labels-is-important"></a></h3>
<p>The <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9372416/">BioLink</a> comes equipped with two important features that are oftem under appreciated: the textual description associated to the node and edge labels, and the hierarchy of the node and edge labels. We can create node type and edge type features from the textual features using procedures such as <a href="https://github.com/AnacletoLAB/grape/blob/main/tutorials/BM25_weighted_pretrained_BERT_based_textual_embedding_using_GRAPE.ipynb">Okapi BM25-weighted BERT</a> and by composing the <a href="https://github.com/LucaCappelletti94/kg-biolink">BioLink hyerarchy into a KG</a> we can use any node embedding model to get additional topological node type and edge type features. These features can be used for many tasks, such as the classification of the nodes and edges, and the prediction of the edges. The <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9372416/">BioLink</a> hierarchy is not perfect, but it provides several good inspirations that should be followed for the Earth Metabolome KG.</p>
<h2 id="todo-tomorrow">Todo tomorrow<a aria-hidden="true" class="anchor-heading icon-link" href="#todo-tomorrow"></a></h2>
<ul>
<li>Delve into whether the data models suggested by <a href="https://www..com/in/chrismungall/">Chris Mungall</a> are a viable solution for the characterization of biological samples.</li>
<li>Start to delve into the <a href="https://github.com/enpkg/enpkg_workflow">ENPKG Workflow</a> so as to plan a holystic refactoring, if needed.</li>
</ul>