# An efficient field and laboratory workflow for plant phylotranscriptomic projects

## Metadata

* Item Type: [[article]]
* Authors: [[ya-yang]], [[michael-j.-moore]], [[samuel-f.-brockington]], [[alfonso-timoneda]], [[tao-feng]], [[hannah-e.-marx]], [[joseph-f.-walker]], [[stephen-a.-smith]]
* Date: [[2017]]
* Date Added: [[2022-04-06]]
* URL: [https://onlinelibrary.wiley.com/doi/abs/10.3732/apps.1600128](https://onlinelibrary.wiley.com/doi/abs/10.3732/apps.1600128)
* DOI: [10.3732/apps.1600128](https://doi.org/10.3732/apps.1600128)
* Cite key: yang_2017
* Tags: #Caryophyllales, #cryogenic-field-sampling, #phylogenomics, #phylotranscriptomics, #RNA, #zotero, #literature-notes, #reference
* PDF Attachments
	- [yang_et_al_2017_an_efficient_field_and_laboratory.pdf](zotero://open-pdf/library/items/3RFSXU2I)

## Abstract

Premise of the study: We describe a field and laboratory workflow developed for plant phylotranscriptomic projects that involves cryogenic tissue collection in the field, RNA extraction and quality control, and library preparation. We also make recommendations for sample curation. Methods and Results: A total of 216 frozen tissue samples of Caryophyllales and other angiosperm taxa were collected from the field or botanical gardens. RNA was extracted, stranded mRNA libraries were prepared, and libraries were sequenced on Illumina HiSeq platforms. These included difficult mucilaginous tissues such as those of Cactaceae and Droseraceae. Conclusions: Our workflow is not only cost effective (ca. $270 per sample, as of August 2016, from tissue to reads) and time efficient (less than 50 h for 10â€“12 samples including all laboratory work and sample curation), but also has proven robust for extraction of difficult samples such as tissues containing high levels of secondary compounds.


##  Zotero links
* [Local library](zotero://select/items/3_JTB4Z36K)
* [Cloud library](http://zotero.org/groups/4613367/items/JTB4Z36K)

