<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width"/><link rel="icon" href="/dendron-dbgi/favicon.ico"/><title>2024-02-12</title><meta name="robots" content="index,follow"/><meta name="googlebot" content="index,follow"/><meta name="description" content="Open Notebook Science in the DBGI"/><meta property="og:title" content="2024-02-12"/><meta property="og:description" content="Open Notebook Science in the DBGI"/><meta property="og:url" content="https://digital-botanical-gardens-initiative.github.io/dendron-dbgi/notes/a5h68k7782sqmjf2eqakaew/"/><meta property="og:type" content="article"/><meta property="article:published_time" content="2/12/2024"/><meta property="article:modified_time" content="2/12/2024"/><link rel="canonical" href="https://digital-botanical-gardens-initiative.github.io/dendron-dbgi/notes/a5h68k7782sqmjf2eqakaew/"/><meta name="next-head-count" content="14"/><link rel="preload" href="/dendron-dbgi/_next/static/css/be4042e32dc1678f.css" as="style"/><link rel="stylesheet" href="/dendron-dbgi/_next/static/css/be4042e32dc1678f.css" data-n-g=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/dendron-dbgi/_next/static/chunks/polyfills-c67a75d1b6f99dc8.js"></script><script src="/dendron-dbgi/_next/static/chunks/webpack-2171ba579db410b3.js" defer=""></script><script src="/dendron-dbgi/_next/static/chunks/framework-28c999baf2863c3d.js" defer=""></script><script src="/dendron-dbgi/_next/static/chunks/main-86951ffaf72f5ad0.js" defer=""></script><script src="/dendron-dbgi/_next/static/chunks/pages/_app-80f57aa65d177734.js" defer=""></script><script src="/dendron-dbgi/_next/static/chunks/935-4dee79e80b8641c6.js" defer=""></script><script src="/dendron-dbgi/_next/static/chunks/6-50972def09142ee2.js" defer=""></script><script src="/dendron-dbgi/_next/static/chunks/pages/notes/%5Bid%5D-78d472fa3b924116.js" defer=""></script><script src="/dendron-dbgi/_next/static/lcvOAFe71opK26LFlnkMt/_buildManifest.js" defer=""></script><script src="/dendron-dbgi/_next/static/lcvOAFe71opK26LFlnkMt/_ssgManifest.js" defer=""></script></head><body><div id="__next" data-reactroot=""><section class="ant-layout" style="width:100%;min-height:100%"><header class="ant-layout-header" style="position:fixed;isolation:isolate;z-index:1;width:100%;border-bottom:1px solid #d4dadf;height:64px;padding:0 24px 0 2px"><div class="ant-row ant-row-center" style="max-width:992px;justify-content:space-between;margin:0 auto"><div style="display:flex" class="ant-col ant-col-xs-20 ant-col-sm-4"></div><div class="ant-col gutter-row ant-col-xs-0 ant-col-sm-20 ant-col-md-20 ant-col-lg-19"><div class="ant-select ant-select-lg ant-select-auto-complete ant-select-single ant-select-allow-clear ant-select-show-search" style="width:100%"><div class="ant-select-selector"><span class="ant-select-selection-search"><input type="search" autoComplete="off" class="ant-select-selection-search-input" role="combobox" aria-haspopup="listbox" aria-owns="undefined_list" aria-autocomplete="list" aria-controls="undefined_list" aria-activedescendant="undefined_list_0" value=""/></span><span class="ant-select-selection-placeholder">For full text search please use the &#x27;?&#x27; prefix. e.g. ? Onboarding</span></div></div></div><div style="display:none;align-items:center;justify-content:center" class="ant-col ant-col-xs-4 ant-col-sm-4 ant-col-md-0 ant-col-lg-0"><span role="img" aria-label="menu" style="font-size:24px" tabindex="-1" class="anticon anticon-menu"><svg viewBox="64 64 896 896" focusable="false" data-icon="menu" width="1em" height="1em" fill="currentColor" aria-hidden="true"><path d="M904 160H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0 624H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8zm0-312H120c-4.4 0-8 3.6-8 8v64c0 4.4 3.6 8 8 8h784c4.4 0 8-3.6 8-8v-64c0-4.4-3.6-8-8-8z"></path></svg></span></div></div></header><section class="ant-layout" style="margin-top:64px;display:flex;flex-direction:row"><div class="site-layout-sidebar" style="flex:0 0 auto;width:calc(max((100% - 992px) / 2, 0px) + 200px);min-width:200px;padding-left:calc((100% - 992px) / 2)"><aside class="ant-layout-sider ant-layout-sider-dark" style="position:fixed;overflow:auto;height:calc(100vh - 64px);background-color:transparent;flex:0 0 200px;max-width:200px;min-width:200px;width:200px"><div class="ant-layout-sider-children"></div></aside></div><main class="ant-layout-content side-layout-main" style="max-width:1200px;min-width:0;display:block"><div style="padding:0 24px"><div class="main-content" role="main"><div class="ant-row"><div class="ant-col ant-col-24"><div class="ant-row" style="margin-left:-10px;margin-right:-10px"><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-24 ant-col-md-18"><div><h1 id="2024-02-12">2024-02-12<a aria-hidden="true" class="anchor-heading icon-link" href="#2024-02-12"></a></h1>
<h1 id="this-is-pmas-dbgi-daily-open-notebook">This is PMA's DBGI daily open-notebook.<a aria-hidden="true" class="anchor-heading icon-link" href="#this-is-pmas-dbgi-daily-open-notebook"></a></h1>
<p>Today is 2024.02.12</p>
<h2 id="todo-today">Todo today<a aria-hidden="true" class="anchor-heading icon-link" href="#todo-today"></a></h2>
<h3 id="have-a-look-at-the-dbgi-discussion-forum">Have a look at the DBGI discussion forum<a aria-hidden="true" class="anchor-heading icon-link" href="#have-a-look-at-the-dbgi-discussion-forum"></a></h3>
<pre><code>- https://github.com/orgs/digital-botanical-gardens-initiative/discussions
</code></pre>
<h3 id=""><a aria-hidden="true" class="anchor-heading icon-link" href="#"></a></h3>
<h3 id="-1"><a aria-hidden="true" class="anchor-heading icon-link" href="#-1"></a></h3>
<h2 id="doing">Doing<a aria-hidden="true" class="anchor-heading icon-link" href="#doing"></a></h2>
<p><a href="https://obo-communitygroup.slack.com/archives/C01P3D02U1H/p1707299939220829">https://obo-communitygroup.slack.com/archives/C01P3D02U1H/p1707299939220829</a></p>
<p>Pierre-Marie Allard
5 days ago
Hello RO people ! Disclaimer: I am very new to RO and the ontology world in general please excuse me in case I say any absurdities.
I would like to find the appropriate RO property to express the fact that a given molecule has been found in a given taxon. In LOTUS (an electronic ressource documenting occurence of small molecules across the tree of life) we have used the Wikidata P703 "found_in_taxon" property. E.g. erythromycin is found in taxon Aeromicrobium erythreum. Now I somehow found that the <a href="https://biolink.github.io/biolink-model/in_taxon/">https://biolink.github.io/biolink-model/in_taxon/</a> LinkML model states that this in_taxon is exactly mapped to RO:0002162 and P703.  Before using RO:0002162 I would like to make sure that there is no better suited property.
One thing that made doubt is the fact that the description states : "Connects a biological entity to its taxon of origin." and that I am here trying to link a chemical entity to its taxon of origin.
The other doubt I had was that "Connects a biological entity to its taxon of origin." could also be interpreted as organism X (for example this specimen defined by an iNaturalist observation) is connected to its taxon of origin in this case Arabidopsis thaliana (55021 or Q158695)
(edited)</p>
<p>eLifeeLife
The LOTUS initiative for open knowledge management in natural products research
The LOTUS initiative builds, through the Wikidata knowledge graph, a virtuous cycle of data sharing practices for natural products research.</p>
<p>wikidata.orgwikidata.org
found in taxon
the taxon in which the item can be found</p>
<p>wikidata.orgwikidata.org
erythromycin
chemical compound</p>
<p>wikidata.orgwikidata.org
Aeromicrobium erythreum
species of bacterium
11 replies</p>
<p>Ray Stefancsik
5 days ago
Have you considered using RO:0002175 present in taxon alone or in combination with other properties?
ebi.ac.ukebi.ac.uk
Ontology Lookup Service (OLS)
OLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions
:+1:
1</p>
<p>Pierre-Marie Allard
5 days ago
No. Until now I haven't. It looks adapted indeed. I like the fact that they are sister properties such as ambiguous_for_taxon, dubious_for_taxon and never_in_taxon. I think we could use the two first one in some cases when documenting the presence of contaminants in a metabolite annotation process for example. This brings me to another question:  Would that be OK to use present_in_taxon for a molecule not biosynthesized by the organism. Could I say acetamiprid (an insecticide) is  present_in_taxon bee (Apis mellifera) ?</p>
<p>ebi.ac.ukebi.ac.uk
Ontology Lookup Service (OLS)
OLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions</p>
<p>ebi.ac.ukebi.ac.uk
Ontology Lookup Service (OLS)
OLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions</p>
<p>ebi.ac.ukebi.ac.uk
Ontology Lookup Service (OLS)
OLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions</p>
<p>ebi.ac.ukebi.ac.uk
Ontology Lookup Service (OLS)
OLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions</p>
<p>wikidata.orgwikidata.org
(E)-acetamiprid
chemical compound</p>
<p>Damion Dooley
4 days ago
You would be doing that at the instance level?  A pesticide is not present in all bees.  Maybe in some bees belonging to a particular hive or area?  So not an “in taxon” relation but rather “part of” some organism or “part of” some organism which is member of some collection.  One can say an organism sample or part is “in taxon” some NCBITaxon entity.  A class-level assertion that would work?: “flight muscle mitochondria present in taxon some Apis mellifera” (edited) </p>
<p>Ray Stefancsik
4 days ago
<a title="Private" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank" class="private">@Pierre-Marie (Private)</a> Allard
, Do you consider the chemical part of some instances of the taxon or want to express that the chemical is located within some instances of the taxon?
It is a valid philosophical question.
erythromycin part of some Aeromicrobium erythreum
erythromycin located in some Aeromicrobium erythreum
I think you could also use RO:0002175 present in taxon in place of located in, but I may be wrong.
ebi.ac.ukebi.ac.uk
Ontology Lookup Service (OLS)
OLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions
ebi.ac.ukebi.ac.uk
Ontology Lookup Service (OLS)
OLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions</p>
<p>Damion Dooley
4 days ago
Ah, right, the question of whether “present in taxon” was intended to cover identity characteristics of a taxon - things that allow us to recognize/classify an entity as belonging to that class (e.g. at stage of life as property’s example shows) - or more loosely allow us to also make claims about an instance of an organism or its part that don’t involve identity, e.g. pacemaker ‘present in taxon’ ‘john doe #2’. It seems that the parent object property “taxonomic class assertion” definition “An assertion that holds between an ontology class and an organism taxon class, which is intepreted to yield some relationship between instances of the ontology class and the taxon.” dissuades us from using it for describing contaminants and “foreign objects”?
I’m interested too in an answer to this modelling question, esp. for contaminants in food.  RO says “located in” is to be used as an instance level relation.  I recall “part of” has been used a lot too at instance level (rather than just to establish class relations). Hmm! (edited) </p>
<p>Chris Mungall
4 days ago
There are various interpretations of molecule class->taxon class:
biosynthesized by pathways encoded by genes in the organism
by product or intermediate of a native pathway of the organism (or further metabolized)
produced by a microbe that is part of the “natural microbiome” [whatever that is] of that organism
is in the metabolome of some tissue sample taken from that organism (guessing this is what LOTUS is?)
Otherwise detected (including Ray’s pesticide case)</p>
<p>Damion Dooley
3 days ago
<a title="Private" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank" class="private">@Chris (Private)</a> Mungall
how would you/RO handle the mereology of contaminants? Doesn’t seem like there’s a ‘has part’ variant reserved for imbedded foreign objects or contaminants that are not contributing to taxa definition at a class level. (edited) </p>
<p>Chris Mungall
3 days ago
can you say more?</p>
<p>Damion Dooley
3 days ago
We can have an instance of say neonicotinoid ‘part of’ an instance of ‘Bombus terrestris’ rather than use ‘present in taxon’ which due to its parent term is supposed to be used at class level.  Yet, ‘part of’ doesn’t discern material that is normally part of the bee; wouldn’t it be informative to say something like ‘exogenous part of’; a relation that works for tooth fillings too etc. (There are ‘has host’ relations, but that’s not quite general enough).</p>
<p>Chris Mungall
3 days ago
What’s the actual owl axiom?</p>
<p>Damion Dooley
5 hours ago
<a title="Private" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank" class="private">@Chris (Private)</a> Mungall
I realize, related to your case 5, that to model a contaminant (rather than biosynthesized molecule) wrt an organism or material, we either need a kind of “X has part Y” relation that indicates the range is a contaminant relative to the domain entity; alternately I guess a “contaminant role” could be assigned between the part and the material, if that’s what you are getting at re. owl axiom. (Like Alotrope’s impurity role).</p>
<p>Answering at</p>
<p>Many thanks for all the very interesting comments !</p>
<p>I see two different issues here (that maybe could be discussed in their own threads ?)</p>
<p><em>Problem 1.</em> <em>Are we linking a molecule to a taxon or to an organism ?</em> The way we have done this for now (at least in the frame of LOTUS) is that we linked the molecule to the lowest taxonomical level (the most precise) of the organism in which it has been found. For example <a href="https://www.wikidata.org/wiki/Q105674316">trigocherrin A</a> <a href="https://www.wikidata.org/wiki/Property:P703">found in taxon</a> <a href="https://www.wikidata.org/wiki/Q3006048">Trigonostemon cherrieri</a> <a href="https://www.wikidata.org/wiki/Property:P248">stated in</a> <a href="https://www.wikidata.org/wiki/Q83059010">"Trigocherrin A, the first natural chlorinated daphnane diterpene orthoester from Trigonostemon cherrieri"</a> (available at <a href="https://sci-hub.se/10.1021/ol2030907">https://sci-hub.se/10.1021/ol2030907</a>). In the original paper there is no information regarding the specific specimen of Trigonostemon cherrieri which was extracted, no geographical coordinate for example.</p>
<p>Thus describing the link to the taxon and not the organism itself is the best we can do (with this type of information source in our hands). So my question is why should we precise <strong>some</strong> as proposed by <a title="Private" href="https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html" target="_blank" class="private">@Ray (Private)</a> ? erythromycin found in taxon <strong>some</strong> Aeromicrobium erythreum (here i replaced the part_of by found_in_taxon so as not to mix the two topics). If the statement is true for a specific organism (e.g. a precise strain of Aeromicrobium erythreum cultivated under given conditions) this still holds true for the taxon Aeromicrobium erythreum and higher taxa ranks (e.g. genus Aeromicrobium). Right ? We are not using <a href="http://purl.obolibrary.org/obo/RO_0002504">always_present_in_taxon</a> so why should we precise <em>some</em> ?</p>
<p>This topic is very interesting for us because in a much greater fashion than genomics, metabolomics reflects the phenotype and is greatly influenced by environmental conditions. So if we take <a href="https://www.wikidata.org/wiki/Q105674316">trigocherrin A</a> <a href="https://www.wikidata.org/wiki/Property:P703">found in taxon</a> <a href="https://www.wikidata.org/wiki/Q3006048">Trigonostemon cherrieri</a> this might be true for this specific tree grown in this part of the island at this period of the year, but will not hold true when the tree is older at another time of the year and in other environmental conditions.
For this researchers should better document the organism they are profiling / extracting (exploiting <a href="https://en.wikipedia.org/wiki/Taxonomic_treatment">taxonomic treatments</a> and possibly ids such as an iNaturalist observation id or other georeferenced specimen e.g. <a href="https://www.gbif.org/occurrence/438823777">https://www.gbif.org/occurrence/438823777</a>). But this is another topic ...</p>
<p><em>Problem 2.</em> <em>What is the nature of the molecule we are linking ?</em> Chris proposed </p>
<ol>
<li>biosynthesized by pathways encoded by genes in the organism</li>
<li>by product or intermediate of a native pathway of the organism (or further metabolized)</li>
<li>produced by a microbe that is part of the “natural microbiome” [whatever that is] of that organism</li>
<li>is in the metabolome of some tissue sample taken from that organism (guessing this is what LOTUS is?)</li>
<li>Otherwise detected (including Ray’s pesticide case)</li>
</ol>
<p>But we could also very well have :</p>
<ol start="6">
<li>exogeneous compound modified by the host</li>
</ol>
<p>or why not</p>
<ol start="7">
<li>exogeneous compound metabolized by a consortium of species from the host microbiota (see <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5534341/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5534341/</a>)</li>
</ol>
<p>Since it is virtually impossible to establish the source of the thousand of molecules characterized during an untargeted metabolomics experiments it would be interesting to find a way to express the following generic statement "this molecule has been detected in an organism identified as belonging to this taxon". We are not saying that they are no other taxon involved in the production of this molecule, nor that it could be a xenobiotic for example, we just can't precise it for the moment.</p>
<p>So if i understand, the <a href="http://purl.obolibrary.org/obo/RO_0002175">present_in_taxon</a> initially proposed by Ray would not be OK or would it ? To me it looked very similar to <a href="https://www.wikidata.org/wiki/Property:P703">found in taxon</a> and quite generic but I must admit I am starting to get lost.</p>
<p>By the way my question initially came from the fact that I am trying to propose a nanopublication template documenting such links across molecules and organisms. See <a href="https://nanodash.knowledgepixels.com/explore?id=https%3A%2F%2Fw3id.org%2Fnp%2FRATGmPlZuuhgKAcqSICT4Qg_J9z5N9rVQbdGt4hJ7yMJM">example</a> for now using  <a href="http://purl.obolibrary.org/obo/RO_0002162">in taxon</a></p>
<p>Happy to elaborate if I have not been clear on some points!</p>
<h4 id="what-part-of-the-plant-should-mix">What part of the plant should mix<a aria-hidden="true" class="anchor-heading icon-link" href="#what-part-of-the-plant-should-mix"></a></h4>
<p>At the moment we take an aliquot (50 mg) and mix only this.
Colin suggested that we could mix the whould sample and THEN aliquot It</p>
<ul>
<li>this means a better representation of the chemistry of the sample</li>
<li>however it also leads to more cross sample contaminations (trhough the powder)</li>
<li>less storage lifetime ?</li>
</ul>
<h4 id="emi-portal-table">EMI portal table<a aria-hidden="true" class="anchor-heading icon-link" href="#emi-portal-table"></a></h4>
<p>Bluetooth
<a href="https://caniuse.com/web-bluetooth">https://caniuse.com/web-bluetooth</a></p>
<p>We should be able to have an app working offline as long as we are logged in.</p>
<p>Edouard mentions the interst of using local maps</p>
<p>me says:
<a href="https://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm">https://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm</a></p>
<p>me says:
<a href="https://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm#SWISSIMAGE-10-cm---Acc%C3%A8s-aux-g%C3%A9odonn%C3%A9es">https://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm#SWISSIMAGE-10-cm---Acc%C3%A8s-aux-g%C3%A9odonn%C3%A9es</a></p>
<p><a href="https://map.geo.admin.ch/?lang=en&#x26;topic=ech&#x26;bgLayer=ch.swisstopo.pixelkarte-farbe&#x26;layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,ch.astra.wanderland-sperrungen_umleitungen&#x26;layers_opacity=1,1,1,0.8,0.8&#x26;layers_visibility=false,false,false,false,false&#x26;layers_timestamp=18641231">https://map.geo.admin.ch/?lang=en&#x26;topic=ech&#x26;bgLayer=ch.swisstopo.pixelkarte-farbe&#x26;layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,ch.astra.wanderland-sperrungen_umleitungen&#x26;layers_opacity=1,1,1,0.8,0.8&#x26;layers_visibility=false,false,false,false,false&#x26;layers_timestamp=18641231</a>,,,,</p>
<p>me says:
<a href="https://github.com/opengisch/QField">https://github.com/opengisch/QField</a></p>
<p>me says:
<a href="https://georust.org/">https://georust.org/</a></p>
<p>me says:
<a href="https://pka.github.io/rust-for-geo/#/title-slide">https://pka.github.io/rust-for-geo/#/title-slide</a></p>
<h2 id="paused">Paused<a aria-hidden="true" class="anchor-heading icon-link" href="#paused"></a></h2>
<h2 id="done">Done<a aria-hidden="true" class="anchor-heading icon-link" href="#done"></a></h2>
<h2 id="notes">Notes<a aria-hidden="true" class="anchor-heading icon-link" href="#notes"></a></h2>
<h2 id="todo-tomorrow-one-day--or-never">Todo tomorrow, one day ... or never<a aria-hidden="true" class="anchor-heading icon-link" href="#todo-tomorrow-one-day--or-never"></a></h2>
<h3 id="-2"><a aria-hidden="true" class="anchor-heading icon-link" href="#-2"></a></h3>
<h3 id="-3"><a aria-hidden="true" class="anchor-heading icon-link" href="#-3"></a></h3>
<h3 id="-4"><a aria-hidden="true" class="anchor-heading icon-link" href="#-4"></a></h3>
<h2 id="today-i-learned-that">Today I learned that<a aria-hidden="true" class="anchor-heading icon-link" href="#today-i-learned-that"></a></h2>
<ul>
<li></li>
</ul></div></div><div style="padding-left:10px;padding-right:10px" class="ant-col ant-col-xs-0 ant-col-md-6"><div><div class=""><div class="ant-anchor-wrapper dendron-toc" style="max-height:calc(100vh - 64px);z-index:1"><div class="ant-anchor"><div class="ant-anchor-ink"><span class="ant-anchor-ink-ball"></span></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#this-is-pmas-dbgi-daily-open-notebook" title="This is PMA&#x27;s DBGI daily open-notebook.">This is PMA&#x27;s DBGI daily open-notebook.</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#todo-today" title="Todo today">Todo today</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#have-a-look-at-the-dbgi-discussion-forum" title="Have a look at the DBGI discussion forum">Have a look at the DBGI discussion forum</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#" title=""></a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#-1" title=""></a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#doing" title="Doing">Doing</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#what-part-of-the-plant-should-mix" title="What part of the plant should mix">What part of the plant should mix</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#emi-portal-table" title="EMI portal table">EMI portal table</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#paused" title="Paused">Paused</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#done" title="Done">Done</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#notes" title="Notes">Notes</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#todo-tomorrow-one-day--or-never" title="Todo tomorrow, one day ... or never">Todo tomorrow, one day ... or never</a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#-2" title=""></a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#-3" title=""></a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#-4" title=""></a></div><div class="ant-anchor-link"><a class="ant-anchor-link-title" href="#today-i-learned-that" title="Today I learned that">Today I learned that</a></div></div></div></div></div></div></div></div></div></div></div><div class="ant-divider ant-divider-horizontal" role="separator"></div><footer class="ant-layout-footer" style="padding:0 24px 24px"></footer></main></section></section></div><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"note":{"id":"a5h68k7782sqmjf2eqakaew","title":"2024-02-12","desc":"","updated":1707750219244,"created":1707722610968,"traitIds":["open-notebook-dbgi-pmallard"],"custom":{},"fname":"open-notebook.dbgi.pmallard.2024.02.12","type":"note","vault":{"fsPath":"vault"},"contentHash":"ab01a5ae6431339a130b5a8f4966cacc","links":[{"type":"wiki","from":{"fname":"open-notebook.dbgi.pmallard.2024.02.12","id":"a5h68k7782sqmjf2eqakaew","vaultName":"vault"},"value":"user.Pierre-Marie","alias":"@Pierre-Marie","position":{"start":{"line":88,"column":1,"offset":4166},"end":{"line":88,"column":14,"offset":4179},"indent":[]},"xvault":false,"to":{"fname":"user.Pierre-Marie"}},{"type":"wiki","from":{"fname":"open-notebook.dbgi.pmallard.2024.02.12","id":"a5h68k7782sqmjf2eqakaew","vaultName":"vault"},"value":"user.Chris","alias":"@Chris","position":{"start":{"line":120,"column":1,"offset":6519},"end":{"line":120,"column":7,"offset":6525},"indent":[]},"xvault":false,"to":{"fname":"user.Chris"}},{"type":"wiki","from":{"fname":"open-notebook.dbgi.pmallard.2024.02.12","id":"a5h68k7782sqmjf2eqakaew","vaultName":"vault"},"value":"user.Chris","alias":"@Chris","position":{"start":{"line":141,"column":1,"offset":7387},"end":{"line":141,"column":7,"offset":7393},"indent":[]},"xvault":false,"to":{"fname":"user.Chris"}},{"type":"wiki","from":{"fname":"open-notebook.dbgi.pmallard.2024.02.12","id":"a5h68k7782sqmjf2eqakaew","vaultName":"vault"},"value":"user.Ray","alias":"@Ray","position":{"start":{"line":154,"column":206,"offset":9085},"end":{"line":154,"column":210,"offset":9089},"indent":[]},"xvault":false,"to":{"fname":"user.Ray"}}],"anchors":{"this-is-pmas-dbgi-daily-open-notebook":{"type":"header","text":"This is PMA's DBGI daily open-notebook.","value":"this-is-pmas-dbgi-daily-open-notebook","line":11,"column":0,"depth":1},"todo-today":{"type":"header","text":"Todo today","value":"todo-today","line":15,"column":0,"depth":2},"have-a-look-at-the-dbgi-discussion-forum":{"type":"header","text":"Have a look at the DBGI discussion forum","value":"have-a-look-at-the-dbgi-discussion-forum","line":17,"column":0,"depth":3},"":{"type":"header","text":"","value":"","line":19,"column":0,"depth":3},"-1":{"type":"header","text":"","value":"-1","line":20,"column":0,"depth":3},"doing":{"type":"header","text":"Doing","value":"doing","line":22,"column":0,"depth":2},"what-part-of-the-plant-should-mix":{"type":"header","text":"What part of the plant should mix","value":"what-part-of-the-plant-should-mix","line":195,"column":0,"depth":4},"emi-portal-table":{"type":"header","text":"EMI portal table","value":"emi-portal-table","line":205,"column":0,"depth":4},"paused":{"type":"header","text":"Paused","value":"paused","line":236,"column":0,"depth":2},"done":{"type":"header","text":"Done","value":"done","line":238,"column":0,"depth":2},"notes":{"type":"header","text":"Notes","value":"notes","line":240,"column":0,"depth":2},"todo-tomorrow-one-day--or-never":{"type":"header","text":"Todo tomorrow, one day ... or never","value":"todo-tomorrow-one-day--or-never","line":242,"column":0,"depth":2},"-2":{"type":"header","text":"","value":"-2","line":244,"column":0,"depth":3},"-3":{"type":"header","text":"","value":"-3","line":245,"column":0,"depth":3},"-4":{"type":"header","text":"","value":"-4","line":246,"column":0,"depth":3},"today-i-learned-that":{"type":"header","text":"Today I learned that","value":"today-i-learned-that","line":249,"column":0,"depth":2}},"children":[],"parent":"vtdgif1ak56c5yc0ckkjg60","data":{}},"body":"\u003ch1 id=\"2024-02-12\"\u003e2024-02-12\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#2024-02-12\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003ch1 id=\"this-is-pmas-dbgi-daily-open-notebook\"\u003eThis is PMA's DBGI daily open-notebook.\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#this-is-pmas-dbgi-daily-open-notebook\"\u003e\u003c/a\u003e\u003c/h1\u003e\n\u003cp\u003eToday is 2024.02.12\u003c/p\u003e\n\u003ch2 id=\"todo-today\"\u003eTodo today\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#todo-today\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003ch3 id=\"have-a-look-at-the-dbgi-discussion-forum\"\u003eHave a look at the DBGI discussion forum\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#have-a-look-at-the-dbgi-discussion-forum\"\u003e\u003c/a\u003e\u003c/h3\u003e\n\u003cpre\u003e\u003ccode\u003e- https://github.com/orgs/digital-botanical-gardens-initiative/discussions\n\u003c/code\u003e\u003c/pre\u003e\n\u003ch3 id=\"\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#\"\u003e\u003c/a\u003e\u003c/h3\u003e\n\u003ch3 id=\"-1\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#-1\"\u003e\u003c/a\u003e\u003c/h3\u003e\n\u003ch2 id=\"doing\"\u003eDoing\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#doing\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003cp\u003e\u003ca href=\"https://obo-communitygroup.slack.com/archives/C01P3D02U1H/p1707299939220829\"\u003ehttps://obo-communitygroup.slack.com/archives/C01P3D02U1H/p1707299939220829\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003ePierre-Marie Allard\n5 days ago\nHello RO people ! Disclaimer: I am very new to RO and the ontology world in general please excuse me in case I say any absurdities.\nI would like to find the appropriate RO property to express the fact that a given molecule has been found in a given taxon. In LOTUS (an electronic ressource documenting occurence of small molecules across the tree of life) we have used the Wikidata P703 \"found_in_taxon\" property. E.g. erythromycin is found in taxon Aeromicrobium erythreum. Now I somehow found that the \u003ca href=\"https://biolink.github.io/biolink-model/in_taxon/\"\u003ehttps://biolink.github.io/biolink-model/in_taxon/\u003c/a\u003e LinkML model states that this in_taxon is exactly mapped to RO:0002162 and P703.  Before using RO:0002162 I would like to make sure that there is no better suited property.\nOne thing that made doubt is the fact that the description states : \"Connects a biological entity to its taxon of origin.\" and that I am here trying to link a chemical entity to its taxon of origin.\nThe other doubt I had was that \"Connects a biological entity to its taxon of origin.\" could also be interpreted as organism X (for example this specimen defined by an iNaturalist observation) is connected to its taxon of origin in this case Arabidopsis thaliana (55021 or Q158695)\n(edited)\u003c/p\u003e\n\u003cp\u003eeLifeeLife\nThe LOTUS initiative for open knowledge management in natural products research\nThe LOTUS initiative builds, through the Wikidata knowledge graph, a virtuous cycle of data sharing practices for natural products research.\u003c/p\u003e\n\u003cp\u003ewikidata.orgwikidata.org\nfound in taxon\nthe taxon in which the item can be found\u003c/p\u003e\n\u003cp\u003ewikidata.orgwikidata.org\nerythromycin\nchemical compound\u003c/p\u003e\n\u003cp\u003ewikidata.orgwikidata.org\nAeromicrobium erythreum\nspecies of bacterium\n11 replies\u003c/p\u003e\n\u003cp\u003eRay Stefancsik\n5 days ago\nHave you considered using RO:0002175 present in taxon alone or in combination with other properties?\nebi.ac.ukebi.ac.uk\nOntology Lookup Service (OLS)\nOLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions\n:+1:\n1\u003c/p\u003e\n\u003cp\u003ePierre-Marie Allard\n5 days ago\nNo. Until now I haven't. It looks adapted indeed. I like the fact that they are sister properties such as ambiguous_for_taxon, dubious_for_taxon and never_in_taxon. I think we could use the two first one in some cases when documenting the presence of contaminants in a metabolite annotation process for example. This brings me to another question:  Would that be OK to use present_in_taxon for a molecule not biosynthesized by the organism. Could I say acetamiprid (an insecticide) is  present_in_taxon bee (Apis mellifera) ?\u003c/p\u003e\n\u003cp\u003eebi.ac.ukebi.ac.uk\nOntology Lookup Service (OLS)\nOLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions\u003c/p\u003e\n\u003cp\u003eebi.ac.ukebi.ac.uk\nOntology Lookup Service (OLS)\nOLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions\u003c/p\u003e\n\u003cp\u003eebi.ac.ukebi.ac.uk\nOntology Lookup Service (OLS)\nOLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions\u003c/p\u003e\n\u003cp\u003eebi.ac.ukebi.ac.uk\nOntology Lookup Service (OLS)\nOLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions\u003c/p\u003e\n\u003cp\u003ewikidata.orgwikidata.org\n(E)-acetamiprid\nchemical compound\u003c/p\u003e\n\u003cp\u003eDamion Dooley\n4 days ago\nYou would be doing that at the instance level?  A pesticide is not present in all bees.  Maybe in some bees belonging to a particular hive or area?  So not an “in taxon” relation but rather “part of” some organism or “part of” some organism which is member of some collection.  One can say an organism sample or part is “in taxon” some NCBITaxon entity.  A class-level assertion that would work?: “flight muscle mitochondria present in taxon some Apis mellifera” (edited) \u003c/p\u003e\n\u003cp\u003eRay Stefancsik\n4 days ago\n\u003ca title=\"Private\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\" class=\"private\"\u003e@Pierre-Marie (Private)\u003c/a\u003e Allard\n, Do you consider the chemical part of some instances of the taxon or want to express that the chemical is located within some instances of the taxon?\nIt is a valid philosophical question.\nerythromycin part of some Aeromicrobium erythreum\nerythromycin located in some Aeromicrobium erythreum\nI think you could also use RO:0002175 present in taxon in place of located in, but I may be wrong.\nebi.ac.ukebi.ac.uk\nOntology Lookup Service (OLS)\nOLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions\nebi.ac.ukebi.ac.uk\nOntology Lookup Service (OLS)\nOLS is a repository for biomedical ontologies that aims to provide a single point of access to the latest ontology versions\u003c/p\u003e\n\u003cp\u003eDamion Dooley\n4 days ago\nAh, right, the question of whether “present in taxon” was intended to cover identity characteristics of a taxon - things that allow us to recognize/classify an entity as belonging to that class (e.g. at stage of life as property’s example shows) - or more loosely allow us to also make claims about an instance of an organism or its part that don’t involve identity, e.g. pacemaker ‘present in taxon’ ‘john doe #2’. It seems that the parent object property “taxonomic class assertion” definition “An assertion that holds between an ontology class and an organism taxon class, which is intepreted to yield some relationship between instances of the ontology class and the taxon.” dissuades us from using it for describing contaminants and “foreign objects”?\nI’m interested too in an answer to this modelling question, esp. for contaminants in food.  RO says “located in” is to be used as an instance level relation.  I recall “part of” has been used a lot too at instance level (rather than just to establish class relations). Hmm! (edited) \u003c/p\u003e\n\u003cp\u003eChris Mungall\n4 days ago\nThere are various interpretations of molecule class-\u003etaxon class:\nbiosynthesized by pathways encoded by genes in the organism\nby product or intermediate of a native pathway of the organism (or further metabolized)\nproduced by a microbe that is part of the “natural microbiome” [whatever that is] of that organism\nis in the metabolome of some tissue sample taken from that organism (guessing this is what LOTUS is?)\nOtherwise detected (including Ray’s pesticide case)\u003c/p\u003e\n\u003cp\u003eDamion Dooley\n3 days ago\n\u003ca title=\"Private\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\" class=\"private\"\u003e@Chris (Private)\u003c/a\u003e Mungall\nhow would you/RO handle the mereology of contaminants? Doesn’t seem like there’s a ‘has part’ variant reserved for imbedded foreign objects or contaminants that are not contributing to taxa definition at a class level. (edited) \u003c/p\u003e\n\u003cp\u003eChris Mungall\n3 days ago\ncan you say more?\u003c/p\u003e\n\u003cp\u003eDamion Dooley\n3 days ago\nWe can have an instance of say neonicotinoid ‘part of’ an instance of ‘Bombus terrestris’ rather than use ‘present in taxon’ which due to its parent term is supposed to be used at class level.  Yet, ‘part of’ doesn’t discern material that is normally part of the bee; wouldn’t it be informative to say something like ‘exogenous part of’; a relation that works for tooth fillings too etc. (There are ‘has host’ relations, but that’s not quite general enough).\u003c/p\u003e\n\u003cp\u003eChris Mungall\n3 days ago\nWhat’s the actual owl axiom?\u003c/p\u003e\n\u003cp\u003eDamion Dooley\n5 hours ago\n\u003ca title=\"Private\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\" class=\"private\"\u003e@Chris (Private)\u003c/a\u003e Mungall\nI realize, related to your case 5, that to model a contaminant (rather than biosynthesized molecule) wrt an organism or material, we either need a kind of “X has part Y” relation that indicates the range is a contaminant relative to the domain entity; alternately I guess a “contaminant role” could be assigned between the part and the material, if that’s what you are getting at re. owl axiom. (Like Alotrope’s impurity role).\u003c/p\u003e\n\u003cp\u003eAnswering at\u003c/p\u003e\n\u003cp\u003eMany thanks for all the very interesting comments !\u003c/p\u003e\n\u003cp\u003eI see two different issues here (that maybe could be discussed in their own threads ?)\u003c/p\u003e\n\u003cp\u003e\u003cem\u003eProblem 1.\u003c/em\u003e \u003cem\u003eAre we linking a molecule to a taxon or to an organism ?\u003c/em\u003e The way we have done this for now (at least in the frame of LOTUS) is that we linked the molecule to the lowest taxonomical level (the most precise) of the organism in which it has been found. For example \u003ca href=\"https://www.wikidata.org/wiki/Q105674316\"\u003etrigocherrin A\u003c/a\u003e \u003ca href=\"https://www.wikidata.org/wiki/Property:P703\"\u003efound in taxon\u003c/a\u003e \u003ca href=\"https://www.wikidata.org/wiki/Q3006048\"\u003eTrigonostemon cherrieri\u003c/a\u003e \u003ca href=\"https://www.wikidata.org/wiki/Property:P248\"\u003estated in\u003c/a\u003e \u003ca href=\"https://www.wikidata.org/wiki/Q83059010\"\u003e\"Trigocherrin A, the first natural chlorinated daphnane diterpene orthoester from Trigonostemon cherrieri\"\u003c/a\u003e (available at \u003ca href=\"https://sci-hub.se/10.1021/ol2030907\"\u003ehttps://sci-hub.se/10.1021/ol2030907\u003c/a\u003e). In the original paper there is no information regarding the specific specimen of Trigonostemon cherrieri which was extracted, no geographical coordinate for example.\u003c/p\u003e\n\u003cp\u003eThus describing the link to the taxon and not the organism itself is the best we can do (with this type of information source in our hands). So my question is why should we precise \u003cstrong\u003esome\u003c/strong\u003e as proposed by \u003ca title=\"Private\" href=\"https://wiki.dendron.so/notes/hfyvYGJZQiUwQaaxQO27q.html\" target=\"_blank\" class=\"private\"\u003e@Ray (Private)\u003c/a\u003e ? erythromycin found in taxon \u003cstrong\u003esome\u003c/strong\u003e Aeromicrobium erythreum (here i replaced the part_of by found_in_taxon so as not to mix the two topics). If the statement is true for a specific organism (e.g. a precise strain of Aeromicrobium erythreum cultivated under given conditions) this still holds true for the taxon Aeromicrobium erythreum and higher taxa ranks (e.g. genus Aeromicrobium). Right ? We are not using \u003ca href=\"http://purl.obolibrary.org/obo/RO_0002504\"\u003ealways_present_in_taxon\u003c/a\u003e so why should we precise \u003cem\u003esome\u003c/em\u003e ?\u003c/p\u003e\n\u003cp\u003eThis topic is very interesting for us because in a much greater fashion than genomics, metabolomics reflects the phenotype and is greatly influenced by environmental conditions. So if we take \u003ca href=\"https://www.wikidata.org/wiki/Q105674316\"\u003etrigocherrin A\u003c/a\u003e \u003ca href=\"https://www.wikidata.org/wiki/Property:P703\"\u003efound in taxon\u003c/a\u003e \u003ca href=\"https://www.wikidata.org/wiki/Q3006048\"\u003eTrigonostemon cherrieri\u003c/a\u003e this might be true for this specific tree grown in this part of the island at this period of the year, but will not hold true when the tree is older at another time of the year and in other environmental conditions.\nFor this researchers should better document the organism they are profiling / extracting (exploiting \u003ca href=\"https://en.wikipedia.org/wiki/Taxonomic_treatment\"\u003etaxonomic treatments\u003c/a\u003e and possibly ids such as an iNaturalist observation id or other georeferenced specimen e.g. \u003ca href=\"https://www.gbif.org/occurrence/438823777\"\u003ehttps://www.gbif.org/occurrence/438823777\u003c/a\u003e). But this is another topic ...\u003c/p\u003e\n\u003cp\u003e\u003cem\u003eProblem 2.\u003c/em\u003e \u003cem\u003eWhat is the nature of the molecule we are linking ?\u003c/em\u003e Chris proposed \u003c/p\u003e\n\u003col\u003e\n\u003cli\u003ebiosynthesized by pathways encoded by genes in the organism\u003c/li\u003e\n\u003cli\u003eby product or intermediate of a native pathway of the organism (or further metabolized)\u003c/li\u003e\n\u003cli\u003eproduced by a microbe that is part of the “natural microbiome” [whatever that is] of that organism\u003c/li\u003e\n\u003cli\u003eis in the metabolome of some tissue sample taken from that organism (guessing this is what LOTUS is?)\u003c/li\u003e\n\u003cli\u003eOtherwise detected (including Ray’s pesticide case)\u003c/li\u003e\n\u003c/ol\u003e\n\u003cp\u003eBut we could also very well have :\u003c/p\u003e\n\u003col start=\"6\"\u003e\n\u003cli\u003eexogeneous compound modified by the host\u003c/li\u003e\n\u003c/ol\u003e\n\u003cp\u003eor why not\u003c/p\u003e\n\u003col start=\"7\"\u003e\n\u003cli\u003eexogeneous compound metabolized by a consortium of species from the host microbiota (see \u003ca href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5534341/\"\u003ehttps://www.ncbi.nlm.nih.gov/pmc/articles/PMC5534341/\u003c/a\u003e)\u003c/li\u003e\n\u003c/ol\u003e\n\u003cp\u003eSince it is virtually impossible to establish the source of the thousand of molecules characterized during an untargeted metabolomics experiments it would be interesting to find a way to express the following generic statement \"this molecule has been detected in an organism identified as belonging to this taxon\". We are not saying that they are no other taxon involved in the production of this molecule, nor that it could be a xenobiotic for example, we just can't precise it for the moment.\u003c/p\u003e\n\u003cp\u003eSo if i understand, the \u003ca href=\"http://purl.obolibrary.org/obo/RO_0002175\"\u003epresent_in_taxon\u003c/a\u003e initially proposed by Ray would not be OK or would it ? To me it looked very similar to \u003ca href=\"https://www.wikidata.org/wiki/Property:P703\"\u003efound in taxon\u003c/a\u003e and quite generic but I must admit I am starting to get lost.\u003c/p\u003e\n\u003cp\u003eBy the way my question initially came from the fact that I am trying to propose a nanopublication template documenting such links across molecules and organisms. See \u003ca href=\"https://nanodash.knowledgepixels.com/explore?id=https%3A%2F%2Fw3id.org%2Fnp%2FRATGmPlZuuhgKAcqSICT4Qg_J9z5N9rVQbdGt4hJ7yMJM\"\u003eexample\u003c/a\u003e for now using  \u003ca href=\"http://purl.obolibrary.org/obo/RO_0002162\"\u003ein taxon\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eHappy to elaborate if I have not been clear on some points!\u003c/p\u003e\n\u003ch4 id=\"what-part-of-the-plant-should-mix\"\u003eWhat part of the plant should mix\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#what-part-of-the-plant-should-mix\"\u003e\u003c/a\u003e\u003c/h4\u003e\n\u003cp\u003eAt the moment we take an aliquot (50 mg) and mix only this.\nColin suggested that we could mix the whould sample and THEN aliquot It\u003c/p\u003e\n\u003cul\u003e\n\u003cli\u003ethis means a better representation of the chemistry of the sample\u003c/li\u003e\n\u003cli\u003ehowever it also leads to more cross sample contaminations (trhough the powder)\u003c/li\u003e\n\u003cli\u003eless storage lifetime ?\u003c/li\u003e\n\u003c/ul\u003e\n\u003ch4 id=\"emi-portal-table\"\u003eEMI portal table\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#emi-portal-table\"\u003e\u003c/a\u003e\u003c/h4\u003e\n\u003cp\u003eBluetooth\n\u003ca href=\"https://caniuse.com/web-bluetooth\"\u003ehttps://caniuse.com/web-bluetooth\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eWe should be able to have an app working offline as long as we are logged in.\u003c/p\u003e\n\u003cp\u003eEdouard mentions the interst of using local maps\u003c/p\u003e\n\u003cp\u003eme says:\n\u003ca href=\"https://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm\"\u003ehttps://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eme says:\n\u003ca href=\"https://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm#SWISSIMAGE-10-cm---Acc%C3%A8s-aux-g%C3%A9odonn%C3%A9es\"\u003ehttps://www.swisstopo.admin.ch/fr/orthophotos-swissimage-10-cm#SWISSIMAGE-10-cm---Acc%C3%A8s-aux-g%C3%A9odonn%C3%A9es\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003e\u003ca href=\"https://map.geo.admin.ch/?lang=en\u0026#x26;topic=ech\u0026#x26;bgLayer=ch.swisstopo.pixelkarte-farbe\u0026#x26;layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,ch.astra.wanderland-sperrungen_umleitungen\u0026#x26;layers_opacity=1,1,1,0.8,0.8\u0026#x26;layers_visibility=false,false,false,false,false\u0026#x26;layers_timestamp=18641231\"\u003ehttps://map.geo.admin.ch/?lang=en\u0026#x26;topic=ech\u0026#x26;bgLayer=ch.swisstopo.pixelkarte-farbe\u0026#x26;layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege,ch.astra.wanderland-sperrungen_umleitungen\u0026#x26;layers_opacity=1,1,1,0.8,0.8\u0026#x26;layers_visibility=false,false,false,false,false\u0026#x26;layers_timestamp=18641231\u003c/a\u003e,,,,\u003c/p\u003e\n\u003cp\u003eme says:\n\u003ca href=\"https://github.com/opengisch/QField\"\u003ehttps://github.com/opengisch/QField\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eme says:\n\u003ca href=\"https://georust.org/\"\u003ehttps://georust.org/\u003c/a\u003e\u003c/p\u003e\n\u003cp\u003eme says:\n\u003ca href=\"https://pka.github.io/rust-for-geo/#/title-slide\"\u003ehttps://pka.github.io/rust-for-geo/#/title-slide\u003c/a\u003e\u003c/p\u003e\n\u003ch2 id=\"paused\"\u003ePaused\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#paused\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003ch2 id=\"done\"\u003eDone\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#done\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003ch2 id=\"notes\"\u003eNotes\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#notes\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003ch2 id=\"todo-tomorrow-one-day--or-never\"\u003eTodo tomorrow, one day ... or never\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#todo-tomorrow-one-day--or-never\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003ch3 id=\"-2\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#-2\"\u003e\u003c/a\u003e\u003c/h3\u003e\n\u003ch3 id=\"-3\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#-3\"\u003e\u003c/a\u003e\u003c/h3\u003e\n\u003ch3 id=\"-4\"\u003e\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#-4\"\u003e\u003c/a\u003e\u003c/h3\u003e\n\u003ch2 id=\"today-i-learned-that\"\u003eToday I learned that\u003ca aria-hidden=\"true\" class=\"anchor-heading icon-link\" href=\"#today-i-learned-that\"\u003e\u003c/a\u003e\u003c/h2\u003e\n\u003cul\u003e\n\u003cli\u003e\u003c/li\u003e\n\u003c/ul\u003e","noteIndex":{"id":"txkHnVTI9AawnquiublXO","title":"The DBGI Dendron notes","desc":"","updated":1688889927268,"created":1645438795591,"custom":{"nav_order":0,"permalink":"/"},"fname":"root","type":"note","vault":{"fsPath":"vault"},"contentHash":"fbac06bb8ff15f08a14f40cc7c9e268e","links":[{"type":"wiki","from":{"fname":"root","id":"txkHnVTI9AawnquiublXO","vaultName":"vault"},"value":"open-notebook","position":{"start":{"line":28,"column":53,"offset":1976},"end":{"line":28,"column":70,"offset":1993},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"open-notebook"}},{"type":"wiki","from":{"fname":"root","id":"txkHnVTI9AawnquiublXO","vaultName":"vault"},"value":"welcome","position":{"start":{"line":34,"column":81,"offset":2595},"end":{"line":34,"column":92,"offset":2606},"indent":[]},"xvault":false,"sameFile":false,"to":{"fname":"welcome"}},{"from":{"fname":"welcome","id":"tofizf0eb38rduuw6ffnzba","vaultName":"vault"},"type":"backlink","position":{"start":{"line":5,"column":27,"offset":59},"end":{"line":5,"column":35,"offset":67},"indent":[]},"value":"root"}],"anchors":{"overview":{"type":"header","text":"Overview","value":"overview","line":8,"column":0,"depth":1},"main-goals":{"type":"header","text":"Main goals","value":"main-goals","line":12,"column":0,"depth":1},"an-open-science-project":{"type":"header","text":"An Open Science project","value":"an-open-science-project","line":18,"column":0,"depth":1},"working-with-the-workshop-doors-open":{"type":"header","text":"Working with the workshop doors open","value":"working-with-the-workshop-doors-open","line":21,"column":0,"depth":2},"open-notebook-science":{"type":"header","text":"Open Notebook Science","value":"open-notebook-science","line":28,"column":0,"depth":2},"whats-new-":{"type":"header","text":"What's new ?!","value":"whats-new-","line":43,"column":0,"depth":2}},"children":["il7c8piddciyg8dw5owj6ui","se2pzun6petcy32hf5j3wuw","bum7d9d6lnc0a863iqaogka","r23kzroton7bo017hsc6lmk","fmlpvd5ebpugd0idxhytk28","iq3hz5b7sskcu56bl8pgbwq","vphzpqwuu98x8sfgfrconej","hl55gsxxo1wlj1igs9mso7p","1sxxmo2bt2eans2xhdorb0q","8yees000uw0bxldzotr8uvw","9p6t4gle5zj5ilkd67j9rud","n9brxvyxekn9b8bv9d8xdik","7ic8gy5h3i6v906dzrig6hy","m0qwxiyv7i2hnkr3fk2m3by","o6r2k35gfsfur2gwgk56ra3","o2psjwj00vl4m8d7rgyfk9f","8u9qb1u3gp4e3osrb73xvor","f7cx4bd1mxjt63d0daza1xb","xoiil9w7pvz3ojhif2a8ppj","o8bu4pl5w2ag7o1h3401qgj","u9u4dax4f23kvq98mtw0ezj","eh30zgccb8kw1z36jlstijv","b17we89hoe3m5ulu9d1t8by","rnoodwuxipqxc8zdxt7kqdm","6c37mounk3i96bftncrvr29","p6xf974t6tqndzis0qjc9j6","165zhtrhauww30lf9dbp8fa","h2jxo1dbamqvqodtuopq9q1","1k6j3afbnh1kiyz95m1kyxz","dwyb49xky735sox320haxga","trzsyfzfrls2if6tjt9dy5u","nnhoqwog50xoiu1bzbn6cyj","q7siltgd71qf2xgppbysw35","tofizf0eb38rduuw6ffnzba","v3eyay0icafeftx6kbyemlm","h8trx40s491czjp1npypo68","eru1s83ybgl3raytc4dgeuz","lykjjec8x2kmqnxjz10nwm6","q8ucho0l7y37kqhprv8ku3k","fynboti1depbjpoz1whdnvt","il17okdfivbzsj1ij63uf8m"],"parent":null,"data":{},"body":"\n# Overview\n\nThe Digital Botanical Gardens Initiative (DBGI, www.dbgi.org) ambitions to explore innovative solutions for the collection, management and sharing of digital information acquired on living botanical collections. A particular focus will be placed on the large scale characterization of the chemodiversity of living plants collections through mass spectrometric approaches. The acquired data will be structured, organized and connected with relevant metadata through [semantic web](https://en.wikipedia.org/wiki/Semantic_Web) technology. The gathered knowledge will then inform ecosystem functioning research and orient [biodiversity conservation](https://en.wikipedia.org/wiki/Conservation_biology) projects. The DBGI initially aims to take advantage of the readily available living collections of Swiss botanical gardens to establish robust and scalable biodiversity digitisation workflows. The ultimate goal is to apply these approaches in the field and at the global scale in wildlife ecosystems. \n\n# Main goals\n\nThe DBGI main goals are briefly summarized below :\n\n![](/assets/images/2022-06-18-17-13-23.png){max-width: 400px, display: block, margin: 0 auto}\n\n# An Open Science project\n\n\n## Working with the workshop doors open\n\n![](/assets/images/2022-06-18-17-47-56.png){max-width: 400px}\n\nA way of working briefly described by Andy Matuschak here https://notes.andymatuschak.org/z21cgR9K3UcQ5a7yPsj2RUim3oM2TzdBByZu. \n\n\n## Open Notebook Science\n\nIf we apply this \"working with the workshop doors open\" idea to a scientific research endeavor we understand it as respecting the core concepts of Open Science. These is more precisely the principle of Open Notebook Science https://en.wikipedia.org/wiki/Open-notebook_science\n\nFor the projects of the DBGI we will follow such an Open Notebook Science approach in order to share early ideas and results but also the less shiny parts of the research projects. \n\nYou can follow the DBGI related Open Notebooks here [[open-notebook]].\n\nExpect to find strange ideas, badly formulated text and poorly written code. However if you have time and interest you should be able to participate at each step of the process. Hopefully this will lead to enhanced collaboration and participation. Maybe yes, maybe not. Let's make it a methodological experiment !\n\nEach pages of this website are written in markdown and hosted on Github. You can edit them by clicking on the \"Click here to edit this page on Github !\" link at the bottom of the page. See down there 👇\n\nIf you are willing to participate the the DBGI Dendron notes have a look at the [[welcome]] section to get you started !\n\n\n## What's new ?!\n\nThis web site might not necessarily seem to be change since the last time you came to visit.\n\nIf something moved recently you should be able to track this over there https://github.com/digital-botanical-gardens-initiative/dendron-dbgi/commits/main"},"collectionChildren":null,"customHeadContent":null,"config":{"version":5,"dev":{"enablePreviewV2":true},"commands":{"lookup":{"note":{"selectionMode":"extract","confirmVaultOnCreate":true,"vaultSelectionModeOnCreate":"smart","leaveTrace":false,"bubbleUpCreateNew":true,"fuzzThreshold":0.2}},"randomNote":{},"insertNote":{"initialValue":"templates"},"insertNoteLink":{"aliasMode":"none","enableMultiSelect":false},"insertNoteIndex":{"enableMarker":false},"copyNoteLink":{"aliasMode":"title"},"templateHierarchy":"template"},"workspace":{"vaults":[{"fsPath":"vault"}],"journal":{"dailyDomain":"daily","name":"journal","dateFormat":"y.MM.dd","addBehavior":"childOfDomain"},"scratch":{"name":"scratch","dateFormat":"y.MM.dd.HHmmss","addBehavior":"asOwnDomain"},"task":{"name":"","dateFormat":"","addBehavior":"childOfCurrent","statusSymbols":{"":" ","wip":"w","done":"x","assigned":"a","moved":"m","blocked":"b","delegated":"l","dropped":"d","pending":"y"},"prioritySymbols":{"H":"high","M":"medium","L":"low"},"todoIntegration":false,"createTaskSelectionType":"selection2link","taskCompleteStatus":["done","x"]},"graph":{"zoomSpeed":1,"createStub":false},"enableAutoCreateOnDefinition":false,"enableXVaultWikiLink":false,"enableRemoteVaultInit":true,"enableUserTags":true,"enableHashTags":true,"workspaceVaultSyncMode":"noCommit","enableAutoFoldFrontmatter":false,"enableEditorDecorations":true,"maxPreviewsCached":10,"maxNoteLength":204800,"dendronVersion":"0.84.0","enableFullHierarchyNoteTitle":false,"enableHandlebarTemplates":true,"hooks":{"onCreate":[{"id":"hello","pattern":"hello.*","type":"js"}]}},"preview":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enableMermaid":true,"enablePrettyRefs":true,"enableKatex":true,"automaticallyShowPreview":false,"enableFrontmatterTags":true,"enableHashesForFMTags":false},"publishing":{"enableFMTitle":true,"enableNoteTitleForLink":true,"enablePrettyRefs":true,"enableKatex":true,"copyAssets":true,"siteHierarchies":["root"],"writeStubs":false,"siteRootDir":"docs","seo":{"title":"DBGI Dendron page","description":"Open Notebook Science in the DBGI"},"github":{"enableEditLink":true,"editLinkText":"Participate and click here to edit this page on Github !","editBranch":"main","editViewMode":"tree","editRepository":"https://github.com/digital-botanical-gardens-initiative/dendron-dbgi"},"enableSiteLastModified":true,"enableFrontmatterTags":true,"enableHashesForFMTags":false,"enableRandomlyColoredTags":true,"enableTaskNotes":true,"enablePrettyLinks":true,"searchMode":"lookup","enableMermaid":true,"enableHierarchyDisplay":true,"assetsPrefix":"/dendron-dbgi","siteUrl":"https://digital-botanical-gardens-initiative.github.io","duplicateNoteBehavior":{"action":"useVault","payload":["vault"]},"ga":{"tracking":"G-EPEXM5GLLG"},"siteFaviconPath":"favicon.ico","siteIndex":"root"}}},"__N_SSG":true},"page":"/notes/[id]","query":{"id":"a5h68k7782sqmjf2eqakaew"},"buildId":"lcvOAFe71opK26LFlnkMt","assetPrefix":"/dendron-dbgi","isFallback":false,"gsp":true,"scriptLoader":[]}</script></body></html>